servers/v3-proxy-api/src/graph/toRest.ts

```diff
@@ -3,17 +3,263 @@
  */
 
 import {
-  GetSavedItemsByOffsetQuery,
-  GetSavedItemsQuery,
+  GetSavedItemsByOffsetCompleteQuery,
+  Tag,
+  Author,
+  DomainMetadata,
+  Image,
+  Video,
+  GetSavedItemsByOffsetSimpleQuery,
   Imageness,
   Videoness,
+  ItemCompleteFragment,
 } from '../generated/graphql/types';
 import {
-  GraphItem,
-  GraphSavedItem,
+  TagsItemObject,
+  AuthorsItemObject,
+  DomainMetadataItemObject,
+  ImagesItemObject,
+  VideosItemObject,
+  VideoTypeMap,
   ListItemObject,
-  RestResponse,
+  ListItemObjectComplete,
+  ListItemObjectAdditional,
 } from './types';
+import { RestResponseSimple, RestResponseComplete } from './types';
+
+/**
```

Comment:
My original version of these reducers was getting too clever with factory functions and generics... Describing them explicitly like this is verbose but I think way more readable.